docs/LogGuide.md

```diff
@@ -1972,21 +2176,30 @@ Parsed Log Line Examples:
 
 <!-- AUTO-GENERATED-CONTENT:END -->
 
-It seems likely that this line was added in order to extend functionality
-for the [NetworkBuff](#line26),
-[NetworkBuffRemove](#line30),
-and [NetworkActionSync](#line37)
-log lines without breaking previous content or plugins.
+This line conveys all current status effects on an entity.
+This can be useful if a plugin or overlay was started after zoning in.
+
+#### Data Fields
+
+This is a variable-length line.
+It can expand up to 30 status effects.
+Beginning with the field called `data3`, each status effect takes three fields.
+
+The first data field for each Trio is some unknown data plus the effect ID in the latter 2 bytes.
+i.e. `field && 0xffff` will get you the effect ID.
+
+The second is the remaining duration as a 32-bit float.
+It comes in as a hex integer, so you will need to do the conversion to float manually.
+For indefinite status effects, this may read out as a fixed value.
+For example, FC buffs will always report a remaining duration of 30 seconds.
+
+The third is the source entity.
 
 <a name="line39"></a>
 
 ### Line 39 (0x27): NetworkUpdateHP
 
-It's not completely clear what triggers this log line,
-but it contains basic information comparable to [NetworkActionSync](#line37) and [NetworkStatusEffects](#line38).
-It applies to allies and fairies/pets.
-
-This log line tends to fire roughly every 3 seconds in some cases.
+This line represents HP/MP ticks.
```

Comment:
I think there's some lost information with this clarification, in that it only applies to players and their pets.

I think that it may also make sense to clarify that the `currentHp` and `currentMp` values from these lines are from the network packet, but other values are from memory and may or may not be stale?

(as an aside, the actual network packet still has a 3rd value where TP used to be, and it does vary, but I've got no idea what it's supposed to represent 🙃 )

Reply:
I guess I could add that back, but I guess bosses wouldn't have it because they just don't have passive HP regen nor MP regen in the first place?

Reply:
I don't think NIN's `Bunshin` has passive HP or MP regen either, but `39` lines still get sent for `Bunshin`:
```
39|2024-05-01T18:34:16.5440000-04:00|40003562|Bunshin|80055|83530|10000|10000|||96.01|103.60|0.03|2.07|c2c46f57b2dfa1fc
```
(not sure what exactly the HP/MP/etc values correspond to for Bunshin, they're not copied from the parent actor at least)

It's entirely possible that server-side, bosses still have a tick just like players and pets, but the server just doesn't send a packet for it because it's not important for the client to have that info.

Reply:
Done